### ISSUE TITLE
Bug - Ascii on Chat Bot

### ISSUE THREAD

**Issue by lightDreaming** [THEREALWWEFAN231/wwe#479]
**WWE version:** 3.0.4
**Minecraft version:** 1.12.2
**Operating system:** Windows
**Optifine/Other mods:** Optifine

**Whats the issue:** If you use ascii's like "█゜" they will not be copied correctly into the chat. They will appear as "ae" as connected letter. 

**Crash-report(Would be very useful also please format):**

**Comment by THEREALWWEFAN231** [comment]
This would be a Minecraft font bug? Maybe not there's a chance the file doesn't read Unicode ill check it out sometime.

**Comment by Lywliety** [comment]
Yeah, I would like to be able to use Unicodes with the chat bot and maybe with the auto sign too (I know you already made it possible by putting an symbol.txt file but it's just not practical at all)
